src/_data/sponsors.json

```diff
@@ -148,6 +148,14 @@
             "monthlyDonation": 200,
             "source": "github",
             "tier": "bronze-sponsor"
+        },
+        {
+            "name": "Nextbase Starter Kit",
+            "image": "https://avatars.githubusercontent.com/u/145838380?v=4",
+            "url": "usenextbase.com",
```

Comment:
Looks like we should append `http://` or `https://` when GitHub returns entry without it, but we could fix that in a follow up.

Reply:
Because as-is it doesn't work on our website. In the preview, it points to `https://deploy-preview-515--new-eslint.netlify.app/sponsors/usenextbase.com`.

Reply:
Ah, the link checker also reports it, so we'll need to fix this to fix the CI.

Reply:
Let me fix it here so the link check works.